README.md

```diff
@@ -30,6 +30,20 @@ Each sample demonstrates one feature of the SDK, together with tests.
 - [**Basic mTLS hello world**](./helloworldmtls): Simple example of a
   Workflow Definition and an Activity Definition using mTLS like Temporal Cloud.
 
+## Building sampes
```

Comment:
What is the use case to build samples instead of just use the instructions in their READMEs to run them?

Reply:
> I don't like following the instruction in the README

This is a bit rough. We need to be able to tell people how to run samples. We can't expect people to not read the README. How do you know what parameters to pass to the executable? How do you know that you may need multiple executables for the sample to work?

> I'd like to have a tool instead, like Make, Mage, or similar.

You do have a tool: `go`. In fact, you're typing _more_ characters and maybe making more mistakes by using `make` to just run `go build` for you followed by running the executable. Just `go run`.